src/App.js

```diff
@@ -139,16 +268,34 @@ class App extends Component {
       return (
         <Naming
           handleName={this.handleName}
-          onConfirm={this.onConfirm}
+          onConfirm={this.onConfirmNames}
         />
       )
+    } else if (this.state.game.gameState === PACKING) {
```

Comment:
`if`, `else if` ... is getting pretty tired in this code. Could be that a `switch` would be more readable here.